docs/visualization/normalization.rst

```diff
@@ -260,3 +260,34 @@ to be used in scripted programs; it's better to use
     ax = fig.add_subplot(1, 1, 1)
     im = ax.imshow(image, origin='lower', norm=norm)
     fig.colorbar(im)
+
+Combining stretches and Matplotlib normalization
+================================================
+
+Stretches can also be combined with other stretches, just like transformations.
+The resulting :class:`~astropy.visualization.stretch.CompositeStretch` can be 
+used to normalize Matplotlib images like any other stretch. For example, a
+composite stretch can stretch residual images with negative values:
+
+.. plot::
+    :include-source:
+    :align: center
+
+    import numpy as np
+    import matplotlib.pyplot as plt
+    from astropy.visualization.stretch import SinhStretch, LinearStretch
+    from astropy.visualization import ImageNormalize
+
+    # Transforms normalized values [0,1] to [-1,1] before stretch and then back
+    stretch = LinearStretch(a=0.5, b=0.5) + SinhStretch() + \
+        LinearStretch(a=2, b=-1)
```

Comment:
Just put this all on one line, it will be more readable and the line length should be ok I think

Reply:
It won't fit on one line with a, b renamed to intercept, slope.